README.md

```
@@ -1,3 +1,82 @@
+
+- 9행, 10열 크기의 게임판을 만든다.
+- 궁(宮) 또는 궁성(宮城)의 영역은 고려하지 않는다.
+    - 가로, 세로 이동만 고려한다.
+
+### 장기 기물
+
+- 장기 기물을 아래의 이미지와 같이 배치해야 한다.    
+  <img src="./img/Janggi_board.png" width="200" height="200">
+
+- 장기 기물 공통 규칙
+    - 이동 경로에 기물이 존재하면 안된다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)